pyquil/_parser/PyQuilListener.py

```diff
@@ -128,11 +128,12 @@ def exitDefCircuit(self, ctx: QuilParser.DefCircuitContext):
         circuit_name = ctx.name().getText()
         variables = [variable.getText() for variable in ctx.variable()]
         qubitVariables = [qubitVariable.getText() for qubitVariable in ctx.qubitVariable()]
+        space = ' ' if qubitVariables else ''
```

Comment:
Not sure if this is the cleanest way to achieve the correct result. The best fix would be to have `Defcircuit` in pyquil.